src/components/POLine/hooks/useFundsWithRestrictedLocationsById/useFundsWithRestrictedLocationsById.js

```diff
@@ -0,0 +1,53 @@
+import {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import { useFundsById } from '../useFundsById';
+import { useGetLocationsByHoldingIds } from '../useGetLocationsByHoldingIds';
+
+export const useFundsWithRestrictedLocationsById = ({
+  fundIDs = [],
```

Comment:
plz, keep consistency in naming with the other places: `fundIds`, `locationIds`, etc.

Reply:
done